api/animal_service/animalServiceModel.js

```diff
@@ -0,0 +1,110 @@
+const db = require('../../data/db-config');
+
+const findAll = async () => {
+  return await db('animal_services as ans')
+    .join('services as s', 's.id', 'ans.service_id')
```

Comment:
Good job with your join statements in this file. They can be hard to keep track of.